README.md

```diff
@@ -2,6 +2,8 @@
 
 # Go beginners workshop
 
+⚠️ ***Please note that the content may not be representative of Ecosia's current engineering standards.***
```

Comment:
This adds a short note to state that this code might not follow our current standards :)